src/Application/Approver.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
Most things that are not presentation or persistence. 

GetApproversWithCategories likely differs quite a bit from optimal design, so don't take it and this new Approver as an example. I purposely ignored issues with GetApproversWithCategories since I figured that on closer inspection there would be reason to refactor it away.